ddl/ingest/checkpoint.go

```diff
@@ -300,6 +291,22 @@ func (s *CheckpointManager) resumeCheckpoint() error {
 	})
 }
 
+func getCurrentInstanceAddr(ctx context.Context, se *sess.Session, serverID uint64) string {
+	template := "select INSTANCE from INFORMATION_SCHEMA.CLUSTER_INFO where SERVER_ID = %d;"
```

Comment:
It can't get the instance info if SEM is enabled

Reply:
Updated. Now we use `config.advertise-address` instead.